Documentation/CodingGuidelines

```diff
@@ -551,10 +551,9 @@ Writing Documentation:
  documentation, please see the documentation-related advice in the
  Documentation/SubmittingPatches file).
 
- In order to ensure the documentation is inclusive, avoid assuming
- that an unspecified example person is male or female, and think
- twice before using "he", "him", "she", or "her".  Here are some
- tips to avoid use of gendered pronouns:
+ In order to ensure the documentation is inclusive, avoid gendering
```

Comment:
avoid verbing a noun for readability especially by non natives.

Reply:
I'll fix the verbing. What do you mean by needless re-flowing?